website/docs/reference/node-selection/syntax.md

```diff
@@ -14,7 +14,7 @@ dbt's node selection syntax makes it possible to run only specific resources in
 | [compile](/reference/commands/compile)     | `--select`, `--exclude`, `--selector`, `--inline`                    |
 | [freshness](/reference/commands/source)    | `--select`, `--exclude`, `--selector`                                |
 | [build](/reference/commands/build)         | `--select`, `--exclude`, `--selector`, `--resource-type`, `--defer`  |
-| [docs generate](/reference/commands/cmd-docs) | `--select`, `--exclude`, `--selector`, `--defer`                  |
+| [docs generate](/reference/commands/cmd-docs) | `--select`, `--exclude`, `--selector`                  |
```

Comment:
@graciegoheen running `dbt docs generate -h` displays `--defer` as an option. Is it there erroneously, or is there another reason for removing it here?

Reply:
I don't think we should callout defer, because it's not actually deferring for the catalog queries. I think we should only callout the selectors relevant to the results returned by the catalog queries.